javascript/javascript3/week2/Week2HW.js

```diff
@@ -0,0 +1,108 @@
+"use strict";
+
+const url =
+  "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json";
+fetch(url)
+  .then((resp) => resp.json())
+  .then((movies) => {
+    const badMovies = movies.filter((movie) => movie.rating < 4);
+    const badMoviesArray = badMovies.map((movieTitle) => movieTitle.title);
+    console.log(badMoviesArray);
+    const badMoviesAfter2000 = badMovies.filter((movie) => movie.year > 2000);
+    const badMoviesAfter2000Array = badMoviesAfter2000.map(
+      (movie) => movie.title
+    );
+    console.log(badMoviesAfter2000Array);
+  });
+
+//Promise that resolves after set time
+
+function promise(resolveAfter) {
+  return new Promise((resolve) => {
+    const delay = resolveAfter * 1000;
+    setTimeout(() => {
+      resolve();
+    }, delay);
+  });
+}
+
+async function result() {
+  console.log("kindly wait");
+  const result = await promise(4);
```

Comment:
You don't have to create `result` variable, because you don't use it. So line 31 can be only
```js
await promise(4);
```

Reply:
I thought we must always use a variable! Good to know, thanks.
and done